Tails specific: gdomap is not used between processes running as the same user
on the same host. No need to stop it properly on shutdown either.

--- chroot.orig/etc/init.d/gdomap	2012-09-24 10:10:20.733885443 +0200
+++ chroot/etc/init.d/gdomap	2012-09-24 10:05:12.829042105 +0200
@@ -6,2 +6,2 @@
-# Default-Start:     2 3 4 5
+# Default-Start:
-# Default-Stop:      0 1 6
+# Default-Stop:
